### ISSUE TITLE
Refactor convicted and acquitted methods on Charge

### ISSUE THREAD

**Issue by wittejm** [codeforpdx/recordexpungPDX#642]
The methods `convicted()` and `acquitted()` on `Charge` should just be based on `DispositionStatus`. 

~~While we're at it, rename `acquitted` to `dismissed` to more closely match the other code and language used in the project. The name `dismissed` isn't quite accurate either because it will include both `dismissed` and `no compaint` charges. This deserves a larger refactor across several project variables names, todo.~~
Edit: All the name refactoring should be its own issue because there a bunch of different things called acquitted/acquittal.

**Comment by wittejm** [comment]
This should have expanded test cases. A simple replacement with
```
    def acquitted(self):
        acquittal_statuses = [DispositionStatus.NO_COMPLAINT, DispositionStatus.DISMISSED]
        return self.disposition and self.disposition.status in acquittal_statuses

    def __convicted(self):
        return self.disposition and self.disposition.status == DispositionStatus.CONVICTED

```
makes all the tests pass. 

The above functions both return `False` if the disposition is Diverted. However, the exunger logic treats acquitted/convicted as a boolean:

https://github.com/codeforpdx/recordexpungPDX/blob/67c4bcc745cb5d4cbbb0c1179a36bf5b7617b5a3/src/backend/expungeservice/expunger/expunger.py#L88

This needs more info from Michael (sent him a message). The Diverted charge either counts as a conviction or an acquittal for the purpose of doing time analysis. 

TODO1: get the correct behavior from Michael and add a test case for a Diverted disposition.  
TODO2: every value of the DispositionStatus enum needs to count as either convicted or dismissed for the purpose of time analysis, except for Unknown. We should enforce that with a new test case.

**Comment by wittejm** [comment]
Closed with #648
